### ISSUE TITLE
Compiling with Android NDK 24 fails for `unable to find library -laaudio`

### ISSUE THREAD

**Issue by ssrlive** [mozilla/rust-android-gradle#90]
It told me that
```
  = note: ld: error: unable to find library -laaudio
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

But I can find `libaaudio` in my NDK folders.

![image](https://user-images.githubusercontent.com/30760636/160271139-5e49a2e5-3610-4367-b7a3-d09eef2b448b.png)

this is [log.txt](https://github.com/mozilla/rust-android-gradle/files/8357105/log.txt) file.

this is my [test project](https://github.com/ssrlive/droid-rust/tree/justatest/native-demo).

**Comment by atsushieno** [comment]
AAudio is available only on Android 8.0 or newer while your project indicates API Level 21, maybe that's the issue.

https://github.com/ssrlive/droid-rust/blob/2071360faefd1ac3e467fcfe4abdebf2eff00bb8/native-demo/example/build.gradle#L31

**Comment by ssrlive** [comment]
you are right. I have fixed it. thanks.

**Comment by ncalexan** [comment]
> AAudio is available only on Android 8.0 or newer while your project indicates API Level 21, maybe that's the issue.
> 
> https://github.com/ssrlive/droid-rust/blob/2071360faefd1ac3e467fcfe4abdebf2eff00bb8/native-demo/example/build.gradle#L31

@atsushieno thanks for suggesting this!  It brings me joy to have an issue answered by a plugin user :)
